### PR TITLE
Implement and style footer

### LINKED ISSUE #23: Implement and style footer
Terms and condition, policy, copyright, etc.

### PR DESCRIPTION
**Pull request by RobPando**
Closes #23

### CODE REVIEW

File: app/views/shared/_footer.html.slim [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.row
+  .col-sm-4
+    p Copyright 2017
```
Comment (pablocrivella) [inline app/views/shared/_footer.html.slim]: Make this a helper

File: app/views/shared/_footer.html.slim [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  .col-sm-4
+    .text-center T
+  .col-sm-4
+    .text-right Terms | Conditions
```
Comment (pablocrivella) [inline app/views/shared/_footer.html.slim]: Move this to translation file

File: app/views/shared/_footer.html.slim (hunk 1 above)
Comment (RobPando) [inline app/views/shared/_footer.html.slim]: make what a helper?, these are just placeholders

File: app/views/shared/_footer.html.slim (hunk 2 above)
Comment (RobPando) [inline app/views/shared/_footer.html.slim]: placeholders, not sure what will go in any of these yet

**Pull request merged**